content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Unified Service Tagging
+kind: documentation
+aliases:
+- /tagging/unified_service_tagging
+further_reading:
+- link: "/tagging/using_tags"
+  tag: "Documentation"
+  text: "Learn how to use tags in the Datadog UI"
+- link: "/blog/autodiscovery-docker-monitoring"
+  tag: "Blog"
+  text: "Learn more about Autodiscovery"
+---
+
+## Overview
+Unified service tagging ties Datadog telemetry together through the use of three [reserved tags][1]: `env`, `service`, and `version`. These tags allow you to correlate related logs and traces by version of a service, trace a spike in CPU usage to corresponding areas of a service, and organize infrastructure more effectively.
+
+The [environment variables][2] `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` form a single point of configuration for all telemetry emitted directly from your service's runtime: [traces][3], [logs][4], and [StatsD metrics][5]. These variables are specific to the runtime of your service. They allow you to define core tagging configuration within your application, rather than across your application and the Agent.
```

Comment:
`These variables are specific to the runtime of your service.` what does this mean exactly? Do we have different environment variable based on the language used?

`They allow you to define core tagging configuration within your application, rather than across your application and the Agent.` Seems to be very APM focused as this is the only place where there is code level configuration.

Could we clarify this by saying that using those environment variable helps to configure all of this once for all data types instead of configuring once for every product?